hpOneView/resources/networking/sas_logical_interconnect_groups.py

```diff
@@ -79,84 +80,4 @@ def get_all(self, start=0, count=-1, filter='', sort='', scope_uris=''):
-
-    def get(self, id_or_uri):
-        """
-        Gets the SAS logical interconnect group.
-
-        Args:
-            id_or_uri: ID or URI of a SAS logical interconnect group.
-
-        Returns:
-            dict: SAS logical interconnect group.
-        """
-        return self._client.get(id_or_uri)
-
-    def get_by(self, field, value):
-        """
-        Gets all SAS logical interconnect groups that match the filter.
-
-        The search is case-insensitive.
-
-        Args:
-            field: Field name to filter.
-            value: Value to filter.
-
-        Returns:
-            list: A list of SAS logical interconnect groups.
-        """
-        return self._client.get_by(field, value)
-
-    def create(self, resource, timeout=-1):
-        """
-        Creates a SAS logical interconnect group.
-
-        Args:
-            resource (dict): Object to create.
-            timeout:
-                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
-                in OneView, just stop waiting for its completion.
-
-        Returns:
-            dict: Created resource.
-
-        """
-        return self._client.create(resource, timeout=timeout, default_values=self.DEFAULT_VALUES)
-
-    def update(self, resource, timeout=-1):
-        """
-        Updates a SAS logical interconnect group.
-
-        Args:
-            resource (dict): Object to update.
-            timeout:
-                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
-                in OneView, just stop waiting for its completion.
-
-        Returns:
-            dict: Updated resource.
-
-        """
-        return self._client.update(resource, timeout=timeout, default_values=self.DEFAULT_VALUES)
-
-    def delete(self, resource, force=False, timeout=-1):
-        """
-        Removes a SAS logical interconnect group. You cannot delete a SAS logical interconnect group when a
-        corresponding SAS logical interconnect is still referencing it. Delete the logical enclosure before
-        attempting to delete the SAS logical interconnect it uses.
-
-        Args:
-            resource: object to delete
-            force:
-                 If set to true, the operation completes despite any problems with
-                 network connectivity or errors on the resource itself. The default is false.
-            timeout:
-                Timeout in seconds. Wait for task completion by default. The timeout does not abort the operation
-                in OneView; it just stops waiting for its completion.
-
-        Returns:
-            bool: Indicates if the resource was deleted successfully.
-
-        """
-        return self._client.delete(resource, force=force, timeout=timeout)
+        return self._helper.get_all(start, count, filter=filter, sort=sort, scope_uris=scope_uris)
```

Comment:
As per spec, we have query param as well. Can you please check it?

Reply:
This query param is not new in this version, its available from API version 300. will add it here.